main/views.py

```diff
@@ -75,39 +75,52 @@ def log_out(request):
 
 # The LoginRequiredMixin it's the same of login_required but for classes
 class SuggestionJobView(LoginRequiredMixin,TemplateView):
-    template_name = 'home-page.html'
-    List1 = Job.objects.filter(category='IT').first()
-    List2 = Job.objects.filter(category="Medical").first()
-    List3 = Job.objects.filter(category='Engineering').first()
-    List4 = Job.objects.filter(category='Design').first()
+    template_name = 'home_page.html'
 
     def get(self,request, *args, **kwargs):
-        model = self.List1
-        model2 = self.List2
-        model3 = self.List3
-        model4 = self.List4
+        job_list = [
+            Job.objects.filter(category='IT').first(),
+            Job.objects.filter(category="Medical").first(),
+            Job.objects.filter(category='Engineering').first(),
+            Job.objects.filter(category='Design').first(),
+        ]
         context = {
-            'model': model,
-            'model2' : model2,
-            'model3' : model3,
-            'model4' : model4,
+            'job_list' : job_list,
             }
         return render(request,self.template_name,context)
 
+    def post(self,request,*args,**kwargs):
+        user_id= UserInformation.objects.get(user=request.user.id)
+        job_id = Job.objects.get(id=request.POST.get('job_id'))
+
+        applied_job = AppliedJob.objects.create(user=user_id,
+                    job=job_id)
```

Comment:
it will be better to write it down as the following:
applied_job = AppliedJob.objects.create(
      user=user_id,
      job=job_id
)
this will make it readable and easy to spot maintain